packages/manager/modules/enterprise-cloud-database/src/enterprise-cloud-database.constants.js

```diff
@@ -0,0 +1,52 @@
+export const ENTERPRISE_CLOUD_DATABASE_STATUS_MAP = {
```

Comment:
Should be in a function in model class like https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/blocks/block.class.js#L11-L22

Reply:
Like this :D

Reply:
This is done